score-lang-cli/src/main/java/org/openscore/lang/cli/SlangCLI.java

```diff
@@ -72,9 +73,15 @@ public String run(
         if (!triggerAsync) {
             StopWatch stopWatch = new StopWatch();
             stopWatch.start();
-            id = scoreServices.triggerSync(compilationArtifact, inputs, systemProperties);
+            if(logValue == null || logValue.equals("DEBUG")) {
+                id = scoreServices.triggerSync(compilationArtifact, inputs, systemProperties, false);
+            }else if(logValue.equals("INFO")){
+                id = scoreServices.triggerSync(compilationArtifact, inputs, systemProperties, true);
+            }else{
+                return "Wrong value for --log. Valid: INFO, DEBUG.";
+            }
             stopWatch.stop();
-            return triggerSyncMsg(id, stopWatch.toString());
+            return (logValue == null || logValue.equals("DEBUG")) ? triggerAsyncMsg(id, compilationArtifact.getExecutionPlan().getName()) : "";
```

Comment:
i do not like the complexity here.. there should be only -Q param in my opinion
it is either quiet or not.. let's not make things complex for no reason